packages/params/src/forkName.ts

```diff
@@ -47,3 +47,9 @@ export type ForkBlobs = Exclude<ForkName, ForkPreBlobs>;
 export function isForkBlobs(fork: ForkName): fork is ForkBlobs {
   return isForkWithdrawals(fork) && fork !== ForkName.capella;
 }
+
+export type ForkPreElectra = ForkPreBlobs | ForkName.deneb;
+export type ForkElectra = Exclude<ForkName, ForkPreElectra>;
```

Comment:
Yes ForkElectra is a little confusing here imo.

Maybe ForkPostElectra.

I'd also be in favor of using that same naming for the other forks (eg ForkPostBellatrix instead of ForkExecution)

Reply:
> I'd also be in favor of using that same naming for the other forks (eg ForkPostBellatrix instead of ForkExecution)

Yea I guess back in the days, each hard fork has a single signature feature that you can name after. For electra, there is a bunch of new features that are significant, so naming convention like `ForkExecution` can't apply to electra.

Reply:
ForkElectraOnwards?

Reply:
in favor of pre / post fork terminology, it seems most commonly used and the most concise